packages/suite/src/actions/wallet/coinjoinAccountActions.ts

```diff
@@ -235,17 +249,27 @@ export const fetchAndUpdateAccount =
                 // TODO accountInfo.utxo don't have proper utxo.confirmations field, only 0/1
 
                 // calculate account anonymity set in CoinjoinClient
-                const accountInfoWithAnonymitySet = await dispatch(
-                    analyzeTransactions(accountInfo, account.symbol),
-                );
+                const result = await dispatch(analyzeTransactions(accountInfo, account.symbol));
+                if (!result) {
```

Comment:
This error could be handled as the other ones. What about https://github.com/trezor/trezor-suite/pull/7231/commits/e2366afcce61cac928b0e69852ce5159b65d2758?

Reply:
fair point, i've created followup issue for that
https://github.com/trezor/trezor-suite/issues/7247